src/extension.ts

```diff
@@ -490,7 +491,7 @@ class ExtensionManager implements vscode.Disposable {
                                              await scanForKitsIfNeeded(cmt);
 
     let should_configure = cmt.workspaceContext.config.configureOnOpen;
-    if (should_configure === null && process.env['CMT_TESTING'] !== '1') {
+    if (should_configure === null && util.envGetValue(process.env, 'CMT_TESTING') !== '1') {
```

Comment:
The names of these new functions and the way we access the environment seem a little weird now.  I wouldn't think to go to `util` to get environment values.  I think it would be better if we had an `environment` or `env` module that managed all of this.

And accessing the `process.env` directly should be ok.  `process.env` looks up environment variables in a [case-insensitive way on Windows](https://nodejs.org/api/process.html#process_process_env) which should be faster than the new function.